NGINX/nginx/tests/json_delete.json

```diff
@@ -0,0 +1,69 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "NGINX",
+        "dialect_uuid": "ab25af2e-4916-40ba-955c-34d2301c1f51"
+      }
+    },
+    "message": "{\"server_name\":\"rdp.acme.com\",\"server_addr\":\"5.6.7.8\",\"remote_addr\":\"1.2.3.4\",\"remote_user\":\"\",\"time_local\":\"02/Jun/2020:13:07:18 +0200\",\"request\":\"DELETE /blobstore/aaa/bbb/530fd9ee-23af-4d4e-a9e4-0b7280ace286/document.tgz HTTP/1.1\",\"status\":\"204\",\"body_bytes_sent\":\"0\",\"http_referer\":\"\",\"http_user_agent\":\"Mozilla/5.0 (Windows NT 6.1; Trident/7.0; rv:11.0) like Gecko\",\"proxy_add_x_forwarded_for\":\"1.2.3.4\",\"request_time\":\"0.007\",\"ssl_protocol\":\"\",\"ssl_cipher\":\"\",\"sent_http_content_type\":\"\"}"
+  },
+  "expected": {
+    "message": "{\"server_name\":\"rdp.acme.com\",\"server_addr\":\"5.6.7.8\",\"remote_addr\":\"1.2.3.4\",\"remote_user\":\"\",\"time_local\":\"02/Jun/2020:13:07:18 +0200\",\"request\":\"DELETE /blobstore/aaa/bbb/530fd9ee-23af-4d4e-a9e4-0b7280ace286/document.tgz HTTP/1.1\",\"status\":\"204\",\"body_bytes_sent\":\"0\",\"http_referer\":\"\",\"http_user_agent\":\"Mozilla/5.0 (Windows NT 6.1; Trident/7.0; rv:11.0) like Gecko\",\"proxy_add_x_forwarded_for\":\"1.2.3.4\",\"request_time\":\"0.007\",\"ssl_protocol\":\"\",\"ssl_cipher\":\"\",\"sent_http_content_type\":\"\"}",
+    "action": {
+      "name": "DELETE",
+      "outcome": "success",
+      "properties": [
+        {
+          "time_local": "02/Jun/2020:13:07:18 +0200",
+          "proxy_add_x_forwarded_for": "1.2.3.4",
+          "request_time": "0.007"
```

Comment:
I suggest using `event.duration` (be carrefully, `event.duration` is in nanoseconds)